src/test/java/edu/cornell/kfs/vnd/batch/service/impl/CuCommodityCodeUpdateServiceImplTest.java

```diff
@@ -1,45 +1,55 @@
-
-import edu.cornell.kfs.vnd.batch.service.CommodityCodeUpdateService;
-
-@ConfigureContext
-public class CuCommodityCodeUpdateServiceImplTest extends KualiTestBase {
-	
-	private CommodityCodeUpdateService commodityCodeUpdateService;
-    private ConfigurationService  kualiConfigurationService;
-
-	private static final String DATA_FILE_PATH = "src/test/resources/edu/cornell/kfs/vnd/fixture/commodityCodeFlatFile.txt";
-    
-	@Override
-	protected void setUp() throws Exception {
-		super.setUp();
-		commodityCodeUpdateService = SpringContext.getBean(CommodityCodeUpdateService.class);
-	}
-	
-	public void testLoadCommodityCodeFile() {
-		byte[] zeroLength = {};
-		ArrayList<CommodityCode> noList = new ArrayList();
-		byte[] noContent = commodityCodeUpdateService.getFileContent("no such file");
-		ArrayList<CommodityCode> noCommodityList = (ArrayList<CommodityCode>) commodityCodeUpdateService.parseCommodityCodeList(zeroLength);
-		byte[] content = commodityCodeUpdateService.getFileContent(DATA_FILE_PATH);
-		ArrayList<CommodityCode> someCommodityCodes = (ArrayList<CommodityCode>) commodityCodeUpdateService.parseCommodityCodeList(content);
-		
-		assertEquals(noContent.length, zeroLength.length);
-		assertEquals(noCommodityList.size(), noList.size());
-		assertNotSame(content.length, 0);
-		assertNotSame(someCommodityCodes.size(), 0);
-		
-		assert(commodityCodeUpdateService.loadCommodityCodeFile(DATA_FILE_PATH));
-	}
+import org.mockito.Mockito;
+
+import edu.cornell.kfs.sys.batch.service.impl.CuBatchInputFileServiceImpl;
+import edu.cornell.kfs.vnd.batch.CommodityCodeInputFileType;
+
+public class CuCommodityCodeUpdateServiceImplTest {
+
+    private CommodityCodeUpdateServiceImpl commodityCodeUpdateService;
+
+    private static final String DATA_FILE_PATH = "src/test/resources/edu/cornell/kfs/vnd/fixture/commodityCodeFlatFile.txt";
+    private static final int NUMBER_OF_RECORDS_IN_DATA_FILE = 53321;
+
+    @Before
+    public void setUp() throws Exception {
+        commodityCodeUpdateService = Mockito.spy(new CommodityCodeUpdateServiceImpl());
+        Mockito.doReturn(true).when(commodityCodeUpdateService).updateCommodityCodes(Mockito.any());
+
+        CuBatchInputFileServiceImpl batchInputFileService = new CuBatchInputFileServiceImpl();
+        commodityCodeUpdateService.setBatchInputFileService(batchInputFileService);
+
+        CommodityCodeInputFileType commodityCodeInputFileType = new CommodityCodeInputFileType();
+        commodityCodeUpdateService.setCommodityCodeInputFileType(commodityCodeInputFileType);
+    }
+
+    @After
+    public void tearDown() {
+        commodityCodeUpdateService = null;
+    }
+
+    @Test
+    public void testParseCommodityCodeList() {
+        byte[] content = commodityCodeUpdateService.getFileContent(DATA_FILE_PATH);
+        ArrayList<CommodityCode> someCommodityCodes = (ArrayList<CommodityCode>) commodityCodeUpdateService.parseCommodityCodeList(content);
```

Comment:
I don't think it's necessary to cast the returned collection to an ArrayList, since the assertions below are only checking the collection's total size.

Reply:
OK, changed